users/views/settings/posting.py

```diff
@@ -15,8 +17,24 @@ class PostingPage(SettingsPage):
             "title": "Expand content warnings",
             "help_text": "If content warnings should be expanded by default (not honoured by all clients)",
         },
+        "preferred_posting_language": {
+            "title": "Default posting language",
+            "help_text": "",
+            "choices": sorted(
+                [
+                    (lang.alpha_2, lang.name)
+                    for lang in pycountry.languages
+                    if hasattr(lang, "alpha_2")
```

Comment:
This is _probably_ good enough for now - I think it's how most of the other clients work - but we might need to add support for separating the various Spanish dialects in future, for example.